lib/gutentag/tag_validations.rb

```diff
@@ -13,6 +13,9 @@ def call
     klass.validates :name,
       :presence   => true,
       :uniqueness => {:case_sensitive => false}
+
+    limit = klass.columns_hash["name"].limit
+    klass.validates_length_of :name, :maximum => limit if limit
```

Comment:
Ah, this is neat 👍 

One thing I'd probably tweak (though I feel like I'm not always on the ball on this) - switch the condition to `if limit.present?` or `unless limit.nil?`. It's just a bit more precise, even though Ruby treats nil as false.